articles/openshift/oshift-faq.md

```diff
@@ -29,15 +32,14 @@ By offering this service, we take on all that complexity so that customers can i
 
 ### Is UKCloud for OpenShift a single-tenant or multi-tenant solution?
 
-UKCloud for OpenShift is built as a single-tenant platform on-top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single tenant isolation with the economics and flexibility of multi-tenant infrastructure.
+UKCloud for OpenShift is built as an isolated single-tenant platform on-top of UKCloud's secure, assured multi-tenant UKCloud for OpenStack IaaS platform, helping to deliver the benefits of single tenant isolation with the economics and flexibility of multi-tenant infrastructure.
 
 ### How is UKCloud for OpenShift billed?
 
 This service comprises of two main chargeable elements:
 
-  - **Foundation Pack -** provides the foundation OpenShift services to support the orchestration and management of the container runtime environments. Each customer environment requires only a single Foundation Pack
-
-  - **Runtime Pack -** Provides the footprint to host customer’s containerised applications and services. Customers can choose to mix and match their Runtime Packs, and Runtime Packs can be called off individually, however UKCloud strongly recommends the initial deployment of two or more Runtime Packs for production environments to provide resilience to your workloads in the event of the failure of a single Runtime Pack.
+  - **Foundation Pack -** providing an initial footprint of 32GiB of RAM, billed by the month with a one-month minimum commitment
+  - **Runtime Pack -** billed by the month with a one-month minimum commitment based upon the amour of additional RAM consumed
```

Comment:
Add an empty line between the two bullet points so that they render correctly in the generated HTML

Reply:
Also this isn't correct - the foundation pack provides the control plane and infrastructure nodes - 32GiB of RAM suggests customers can run their applications on that 32GiB which they can't.

Reply:
```suggestion

  - **Runtime Pack -** billed by the month with a one-month minimum commitment based upon the amount of additional RAM allocated
```